manager_42.py

```diff
@@ -51,19 +51,23 @@ def __init__(self, from_prj, caching = True):
         self.caching = caching
         self.apiurl = osc.conf.config['apiurl']
         self.project_preference_order = [
-                #'SUSE:SLE-12-SP2:Update',
+                'SUSE:SLE-12-SP3:GA',
+                'SUSE:SLE-12-SP2:Update',
                 'SUSE:SLE-12-SP2:GA',
                 'SUSE:SLE-12-SP1:Update',
                 'SUSE:SLE-12-SP1:GA',
                 'SUSE:SLE-12:Update',
                 'SUSE:SLE-12:GA',
+                'openSUSE:Leap:42.2:Update',
+                'openSUSE:Leap:42.2',
+                'openSUSE:Leap:42.2:NonFree:Update',
+                'openSUSE:Leap:42.2:NonFree',
                 'openSUSE:Leap:42.1:Update',
                 'openSUSE:Leap:42.1',
                 'openSUSE:Leap:42.1:NonFree:Update',
                 'openSUSE:Leap:42.1:NonFree',
                 'openSUSE:Factory',
                 'openSUSE:Factory:NonFree',
-                'openSUSE:Leap:42.2:SLE-workarounds'
```

Comment:
we don't have SLE-workarounds in 42.3 now, we can add it later when we created it...